en/views/helpers/form.rst

```diff
@@ -1540,8 +1540,7 @@ example::
     ]);
 
     // Create a radio set with our custom wrapping div.
-    echo $this->Form->radio('User.email_notifications', [
-        'options' => ['y', 'n'],
+    echo $this->Form->radio('User.email_notifications', ['y', 'n'], [
         'type' => 'radio'
```

Comment:
No, its not necessary @ravage84.  Its been removed already as you can see here - http://book.cakephp.org/3.0/en/views/helpers/form.html#list-of-templates